wagtailio/static/sass/components/_nav-cta.scss

```diff
@@ -93,8 +93,8 @@
         position: absolute;
         bottom: 30px;
         right: 20px;
-        stroke: $color--white;
-        stroke-width: 1px;
+        stroke: currentColor;
+        stroke-width: 4px;
```

Comment:
Here we're affecting designs in a non-contrast mode, we need to add media query to target forced-colors only

Reply:
The `stroke: currentColor` does not seem to have any effect on the regular rendering. In my opinion the `stroke-width` increase would be an improvement to the base design, it's not my place to decide that, so I agree with Albina's suggestion to move these rules into a media query.

Reply:
Thanks @Scotchester @albinazs - I've moved the use of `currentColor` into a media query, but left `stroke-width` because this is part of another minor issue in the audit. I think it would be better to keep the increased width so that the nav CTA is more distinguishable as a link.

Reply:
Ah, gotcha! If we are considering the thin stroke an accessibility issue as well, then I am happy to have the change :)